src/cryptoadvance/specter/services/swan/client.py

```diff
@@ -284,13 +266,10 @@ def update_autowithdrawal_addresses(
                         "clientId": "specter-dev"
                     },
                     "specter_wallet_alias": "seedsigner_demo"
-                },
```

Comment:
`"btcAddresses": []` seems to be disappearing all over. I don't understand why?

Reply:
Pablo changed the return data at some point so we no longer get the list of addresses associated with a Swan wallet. We don't need it anyway, as we can always just overwrite the current with a new autowithdrawal call.